CONTRIBUTING.md

```diff
@@ -45,6 +45,14 @@ To run code linter, use:
 npm run lint
 ```
 
+## API guidelines
+
+When authoring new API methods, consider the following:
+- expose as little as needed. When in doubt, don't expose
```

Comment:
expose as little information as needed. When in doubt, don’t expose new information.

Reply:
Done.